docker-compose/jaeger-docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '2'
+version: '3'
```

Comment:
this can break xdocks in other clients. But am +1 on version 3.

Reply:
yeah, I am aware that it will break other clients. We don't need to bump it if we don't have a use for the healthcheck, which was added in 2.1

Reply:
with version `3` we could perhaps also use configs (something like configmap in k8s) to split templates for ES and C*.